src/constants.py

```diff
@@ -25,7 +25,7 @@
 TLS_SSL_CERT_FILE = "custom-server-cert.pem"
 MYSQL_EXPORTER_PORT = 9104
 CHARMED_MYSQL_SNAP_NAME = "charmed-mysql"
-CHARMED_MYSQL_SNAP_REVISION = 96  # MySQL v8.0.35
+CHARMED_MYSQL_SNAP_REVISION = 97  # MySQL v8.0.36
```

Comment:
TODO: update the revision to official snap build of 8.0.36

Reply:
Updated to revision 98 in [7345d44](https://github.com/canonical/mysql-operator/pull/410/commits/7345d44a5385c734dd3e419a2dfbc742b8887d9c)